app/helpers/host_helper/textual_summary.rb

```diff
@@ -375,13 +375,8 @@ def textual_compliance_status
       compliant = @record.last_compliance_status
       date      = @record.last_compliance_timestamp
       h[:image] = compliant ? "check" : "x"
-      h[:value] = if !compliant
-                    _("Non-Compliant as of %{time} Ago") %
-                    {:time => time_ago_in_words(date.in_time_zone(Time.zone)).titleize}
-                  else
-                    _("Compliant as of %{time} Ago") %
-                    {:time => time_ago_in_words(date.in_time_zone(Time.zone)).titleize}
-                  end
+      h[:value] = compliant ? _("Compliant as of %{time} Ago") : _("Non-Compliant as of %{time} Ago") %
```

Comment:
@ZitaNemeckova This will not work correctly, you need to wrap the expression inside one more brackets, like this:

```ruby
(compliant ? _("Compliant as of %{time} Ago") : _("Non-Compliant as of %{time} Ago")) % 
{:time => time_ago_in_words(date.in_time_zone(Time.zone)).titleize}
```